extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -256,6 +257,9 @@ public function getWikiVariables() {
 
 		$this->response->setVal( 'data', $wikiVariables );
 		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		//cache wikiaVariables for 1 minute		
+		$this->response->setCacheValidity( self:: WIKIA_VARIABLES_CACHE_TTL);
```

Comment:
Can we cache it for a bit longer time, like an hour or a day? And invalidate the cache when required.